test.py

```diff
@@ -1,9 +1,24 @@
 import unittest
-from preprocess import read_text
+import preprocess as p
 
 class MainTestCase(unittest.TestCase):    
     def io_test(self):
-        self.assertEqual('Our team had a debate for best names for looping variables ? i won', read_text('text_files/test.txt'))
+        self.assertEqual('Our team had a debate for best names for looping variables ? i won', p.read_text('text_files/test.txt'))
+
+    def test_text_proc_replace_contractions(self):
+        test_string = "Hi   \nHow're\nYou?\nI'm well!\n Don't? Would've, I'll, she's, she'll!\n"
+        expected_string_no_conts = "hi how are you i am well do not would have i will she is she will"
```

Comment:
These are hard to read, but I can't think of any other way of doing this, and it makes sense!

Reply:
I tried to pick a complicated test case full of new lines, capital letters, and contractions - I didn't try to make it make sense, but I agree that if I had a good example that actually made sense that would've been better!